best-practice/definitions.ts

```diff
@@ -9,6 +9,57 @@ const github: readonly IBestPractice[] = [
     howToCheck: "Manual. View the repository on https://github.com",
     howToExempt: "N/A",
   },
+  {
+    name: "Branch Protection",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Enable branch protection for the default branch, ensuring changes are reviewed before being deployed.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Access",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Grant access on a team basis, rather than directly to individuals.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Collaborators",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Include at least one GitHub team as a collaborator with admin access.",
```

Comment:
I was surprised to see the terminology 'collaborator' here, as in the past with GitHub repos I've taken 'collaborator' to mean specifically 'outside collaborator' (accounts from outside the Guardian) - I think that's how the term was perhaps originally used in the GitHub UI? Checking some GitHub docs tho, it does seem that 'collaborator' is now used collectively for people both inside and outside of the organisation, [at least as for as the GitHub API goes](https://docs.github.com/en/rest/collaborators/collaborators?apiVersion=2022-11-28#list-repository-collaborators).

In any case, this rule might be better named 'Admin Access' or something like that - I think the key part is we want to make sure _admin_ access is granted to a team?

Reply:
Great point! We say an admin group owns the repository; 'Admin Access' or 'Ownership' are really sensible renames. Want to raise a PR?

Reply:
Done with https://github.com/guardian/recommendations/pull/134!